src/types/config.ts

```diff
@@ -4,7 +4,8 @@ interface RelatedConfig {
   showOnPlaybackDone: boolean;
   showOnPlaybackPaused: boolean;
   playlistId: string | null;
```

Comment:
playlistId should be ProviderPlaylistInfoObject see https://github.com/kaltura/kaltura-player-js/blob/master/src/kaltura-player.js#L191

Reply:
updated